app/assets/javascripts/oxalis/model.js

```diff
@@ -279,7 +279,7 @@ export class OxalisModel {
     this.connectionInfo = new ConnectionInfo();
     this.binary = {};
     for (const layer of layers) {
-      const maxLayerZoomStep = Math.log(Math.max(..._.flatten(layer.resolutions))) / Math.LN2;
+      const maxLayerZoomStep = Math.pow(2, layer.resolutions.length - 1);
```

Comment:
Why do we want that? I would expect the `max` version to be more robust in case individual layers are missing. If we do this, we should add a check (probably in the backend, that there are no gaps and resolutions start with 0 / 1)

Reply:
I think git showed you an outdated diff there. I’m using the logarithm method again.

Reply:
No outdated diff, I went to your changes commit by commit to avoid reviewing my own changes.
I left on minor suggestion regarding regexs, PR LGTM